package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "code-oss-dev",
   "version": "1.73.0",
-  "distro": "8b662eef9d11353c60c0745050c14dfd4918733e",
+  "distro": "5148b57a80a6b4e46339f05ac62e7ca3e0929e33",
```

Comment:
How to generate this string?

Reply:
This is a commit hash pointer to our distro repository. So it is generated by git in that other repo.